gravitee-apim-portal-webui-next/src/components/subscribe/plan-card/plan-card.component.scss

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@use '../../../scss/theme';
+
+.plan-card {
+  --mdc-radio-state-layer-size: 0;
+
+  min-height: 220px;
+  cursor: pointer;
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__header-content {
+    display: flex;
+    align-items: center;
+    margin: 0 auto 10px 0;
+  }
+}
+
+.plan-card.selected {
+  border: 1px solid hsl(344deg 27% calc(61% * 0.8));
```

Comment:
Shouldn't the color come from the theme?

Reply:
Fixed